src/libcore/ops.rs

```diff
@@ -445,7 +445,10 @@ rem_impl! { usize u8 u16 u32 u64 isize i8 i16 i32 i64 }
 impl Rem for f32 {
     type Output = f32;
 
-    // see notes in `core::f32::Float::floor`
+    // The builtin f32 frem doesn't compile when targeting
+    // msvc; use f64 rem instead.
+    // FIXME: This is a bug; trans should be fixed instead
+    // of working around it here.
```

Comment:
Given that the intrinsic isn't actually explicitly referenced here, pointing at floor() would be confusing at best.  Also, it's impossible to fix #27859 here.